jenkins/e2e-image/e2e-runner.sh

```diff
@@ -411,7 +411,7 @@ if [[ -n "${BOOTSTRAP_MIGRATION:-}" ]]; then
   # until that time. This environment variable will do that.
   source "$(dirname "${0}")/upload-to-gcs.sh"
   version=$(find_version)  # required by print_started
-  print_started | jq '.metadata?' > "${ARTIFACTS}/metadata.json"
+  print_started | jq '.metadata? + {version, "job-version"}' > "${ARTIFACTS}/metadata.json"
```

Comment:
will this line blow up if we stop including `version` or `job-version` in the output of `print_started`?